tests/rbd/test_rbd_persistent_writeback_cache_flush.py

```diff
@@ -0,0 +1,124 @@
+"""RBD Persistent write back cache, Test concurrent writes to same image
+
+Test Case Covered:
+CEPH-83574893 - Verify cache flush command when IO is stopped abruptly
+with persistent cache enables via same or different IO tools in SSD mode
+
+Steps :
+1) Setup persistent write cache for an image
+2) Write data to the image using fio jobs
+3) Stop fio abruplty and verify that cache flush command
+
+Environment and limitations:
+ - The cluster should have 5 nodes + 1 SSD cache node
+ - cluster/global-config-file: config/quincy/upi/octo-5-node-env.yaml
+ - Should be Bare-metal.
+
+Support
+- Configure cluster with PWL Cache.
+- Only replicated pool supported, No EC pools.
+"""
+import re
+from time import sleep
+
+from ceph.parallel import parallel
+from ceph.utils import get_node_by_id
+from tests.rbd.rbd_peristent_writeback_cache import (
+    PersistentWriteAheadLog,
+    fio_ready,
+    get_entity_level,
+)
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def kill_fio(rbd):
+    """
+    Kill the fio process running on the client
+    """
+    sleep(60)
```

Comment:
why we need to sleep here?

Reply:
So the idea here is that we start writing IOs and kill the fio process abruptly in between and check the operation of cache flush command.
Since we need to stop IOs after a few seconds after starting them I have added the sleep here. Hope that's clear? @sunilangadi2